client/layout/logged-out.jsx

```diff
@@ -79,6 +80,7 @@ const LayoutLoggedOut = ( {
 		sectionName === 'checkout' && currentRoute.startsWith( '/checkout/jetpack/thank-you' );
 
 	const isReaderTagPage = sectionName === 'reader' && pathNameWithoutLocale.startsWith( '/tag/' );
+	const isReaderTagEmbed = isReaderTagPage && isReaderTagEmbedPage( window.location );
```

Comment:
nit: you probably don't need isReaderTagPage since that function checks for /tag/ in the path right?

Reply:
Agree, I pushed in a change to fix this.